build.sh

```diff
@@ -37,6 +37,11 @@ case $ARCH in
 		TARGETARCH=
 		DOCKERFILE=Dockerfile
 		;;
+	s390x)
+	  	ARCH=s390x
+	  	BUILDARCH=s390x
+		TARGETARCH=
+		DOCKERFILE=Dockerfile-s390x
```

Comment:
Need to add a `;;` on the next line after this

Reply:
Thank you!